package.json

```diff
@@ -24,11 +24,11 @@
   "author": "Gregor Martynus (https://twitter.com/gr2m)",
   "license": "MIT",
   "dependencies": {
-    "@octokit/types": "^9.3.0"
+    "@octokit/types": "^10.0.0"
   },
   "devDependencies": {
     "@gimenete/type-writer": "^0.1.5",
-    "@octokit/core": "^4.0.0",
+    "@octokit/core": "^5.0.0-beta.3",
```

Comment:
do you want to merge with the beta release or wait until 5.0.0 is out?

Reply:
It's going to be merged into the `beta` branch anyways, so it doesn't matter

Reply:
ha true sorry